README.md

```diff
@@ -38,10 +38,10 @@ docker run -v /var/run/docker.sock:/var/run/docker.sock \
            -a host="$(uname -n)"
 ```
 
-You can supply log, stats and event tokens using environment variables:
-`INSIGHT_LOGSTOKEN`: Log token for logs
-`INSIGHT_STATSTOKEN`: Log token for stats
-`INSIGHT_EVENTSTOKEN`: Log token for events
+You can supply log, stats and event tokens using environment variables:  
+`INSIGHT_LOGSTOKEN`: Log token for logs  
```

Comment:
What? Spaces? How does that fix the formatting?

Reply:
Keep in mind Markdown parsers all like to do their own thing. But if you want this to be parsed correctly, use bullet points ;)

Reply:
Double space in markdown produces a newline. Enter doesn't

Reply:
good idea

Reply:
Spaces at the end of a line get ignored by Markdown afaik :P

Reply:
Nope, learned that ages ago. Previewing the README produces the newlines, but you're right better to be explicit with bullet points